.github/workflows/pr.yaml

```
@@ -24,6 +25,31 @@ jobs:
       - devcontainer
     secrets: inherit
     uses: rapidsai/shared-workflows/.github/workflows/pr-builder.yaml@branch-24.12
+    if: always()
+    with:
+      needs: ${{ toJSON(needs) }}
+  changed-files:
+    secrets: inherit
+    uses: rapidsai/shared-workflows/.github/workflows/changed-files.yaml@branch-24.12
+    with:
+      files_yaml: |
+        test_cpp:
+          - '**'
+          - '!.devcontainer/**'
+          - '!.pre-commit-config.yaml'
+          - '!CONTRIBUTING.md'
+          - '!README.md'
+          - '!python/**'
+          # TODO: Remove this before merging
+          - '!.github/workflows/**'
+        test_python:
+          - '**'
+          - '!.devcontainer/**'
+          - '!.pre-commit-config.yaml'
+          - '!CONTRIBUTING.md'
+          - '!README.md'
+          # TODO: Remove this before merging
+          - '!.github/workflows/**'
```

```suggestion
          - '!README.md'
          - '!img/**'
          - '!python/**'
          # TODO: Remove this before merging
          - '!.github/workflows/**'
        test_python:
          - '**'
          - '!.devcontainer/**'
          - '!.pre-commit-config.yaml'
          - '!CONTRIBUTING.md'
          - '!README.md'
          - '!img/**'
          # TODO: Remove this before merging
          - '!.github/workflows/**'
```

The `img/` folder seems to just contain static images for the docs, so I think it's safe to assume that changes only touching that folder don't require re-runs of the Python or C++ tests.

---

Ahh, I don't know how I missed that this one had `img/`. I will add it.